preprocessors/celebrity.schema.json

```
@@ -0,0 +1,34 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/preprocessors/celebrity-detector.schema.json",
+    "type": "object",
+    "title": "Celebrity Detector",
+    "description": "Preprocessor to determine detect celebrities in an image. The associated information can be found in preprocessor named 'ca.mcgill.a11y.image.preprocessor.celebrityDetector'. ",
+    "properties":{
+        "celebrity":{
```

Also: require this key.
```suggestion
        "celebrities":{
```